onoff.js

```diff
@@ -5,8 +5,13 @@ const debounce = require('lodash.debounce');
 const Epoll = require('epoll').Epoll;
 
 const GPIO_ROOT_PATH = '/sys/class/gpio/';
+
+// fs reads return char data
 const ZERO = Buffer.from('0');
```

Comment:
If `GPIO_ZERO` is renamed to `LOW` (see below) then `ZERO` should be renamed to `LOW_BUF`.